include/EventComponent.hpp

```diff
@@ -24,15 +24,22 @@ class EventComponent {
     void addHandler();
 
 private:
-    std::unordered_map<int64_t, void(*)(Ecs& ecs, const EntityId&, void*)> _handlers;
+    std::unordered_map<int64_t, void(*)(Ecs& ecs, const EntityId&, void*)>          _handlers;
+    std::unordered_map<int64_t, void(*)(Ecs& ecs, const EntityId&, void*, void*)>   _logicHandlers;
```

Comment:
What are the void pointer parameters? They probably could be named. Also the function pointer prototype could be a typedef.